src/settings.ts

```diff
@@ -305,6 +330,41 @@ export const BURN_VENUE_TEMPLATES: Array<Template> = [
   },
 ];
 
+// @debt Refactor this constant into types/templates or similar?
+// NOTE: The Map object holds key-value pairs and remembers the original insertion order of the keys
+// Same can be accomplished by a regular object if created through Object.fromEntries()
+export const BURN_VENUE_TEMPLATES_MAP: Map<VenueTemplate, Template> = new Map(
+  BURN_VENUE_TEMPLATES.map((item) => [item.template, item])
+);
+
+export const SPACES_VENUES_AND_ROOMS: VenueRoom[] = [
```

Comment:
Can you simplify this name? Maybe VENUE_ROOMS or VENUE_SPACES

Reply:
The issue is of already simplified names in `settings.ts` and in types in general having `venueRooms` but representing different concepts.

There is a need of one name representing both room and venue, like `SPACES_PORTALS` if portal could be superset of venue and room. But as of now, there are few settings already dealing with venues/rooms/templates and arrays of in that file that untangling those would require separate PR (there are a lot of `@debt` comments).

If portal is acceptable, I could use it or some other single term doing the same function.

Reply:
portal is acceptable, and most of the product team already refers to these as `portals`

Reply:
renamed in 5d5e8b9982703f31a2fb7b8e15ad090912057cdd